contracts/token/ERC1155/extensions/ERC1155Pausable.sol

```diff
@@ -36,7 +41,9 @@ abstract contract ERC1155Pausable is ERC1155, Pausable {
         uint256[] memory amounts,
         bytes memory data
     ) internal virtual override {
-        require(!paused(), "ERC1155Pausable: token transfer while paused");
+        if (paused()) {
+            revert ERC1155Paused();
```

Comment:
I thought we agreed on using the modifier from pausable here ?

Reply:
This is correct, I forgot. Just updated.